gfx/video_crt_switch.c

```diff
@@ -0,0 +1,165 @@
+/* CRT SwitchRes Core 
+ * Copyright (C) 2018 Ben Templeman.
+ *
+ * RetroArch - A frontend for libretro.
+ *  Copyright (C) 2010-2014 - Hans-Kristian Arntzen
+ *  Copyright (C) 2011-2017 - Daniel De Matteis
+ *
+ *  RetroArch is free software: you can redistribute it and/or modify it under the terms
+ *  of the GNU General Public License as published by the Free Software Found-
+ *  ation, either version 3 of the License, or (at your option) any later version.
+ *
+ *  RetroArch is distributed in the hope that it will be useful, but WITHOUT ANY WARRANTY;
+ *  without even the implied warranty of MERCHANTABILITY or FITNESS FOR A PARTICULAR
+ *  PURPOSE.  See the GNU General Public License for more details.
+ *
+ *  You should have received a copy of the GNU General Public License along with RetroArch.
+ *  If not, see <http://www.gnu.org/licenses/>.
+*/
+#include <stddef.h>
+#include <string.h>
+#include <stdlib.h>
+
+#include <gfx/video_driver.h>

```

Comment:
This is invalid (however gcc doesn't complain) and fails completely on MSVC with an error, use `#include "video_driver.h"` instead, and these CRT files (the OS-specific ones) need to go in their own directory.

Reply:
I thought I had already done this, it turns out I did not.  This is done